client/state/selectors/is-connected-secondary-network-site.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { isMainSiteToSecondarySiteConnection } from 'state/selectors';
+
+/**
+ * Returns true if site with id equal to siteId is a connected secondary network site and false otherwise
+ *
+ * @param  {Object}  state       Global state tree
+ * @param  {Number}  siteId      The ID of the site we're querying
+ * @return {?Boolean}            Whether site with id equal to siteId is a connected secondary network site
```

Comment:
`some` will always return a Boolean value, so let's remove the `null` notation from the JSdoc.